<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHImageDebug</class>
 <widget class="QDockWidget" name="QHImageDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>259</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QHImageDebug</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DVP PCLK Phase Shift(Dec):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dvpPhaseShiftcomboBox">
        <item>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">270</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="phaseShiftpushButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="fpgaDecodeInfolabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="infolabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
